External/HowTo/README.md

```diff
@@ -37,11 +37,11 @@ You might have had the OnSelect function or some other method name responding to
 2. In that MonoBehavior, implement the interface called IInputClickHandler. This will get raised when a tap is detected by GesturesInput.
 3. The event data inside the OnInputClicked event handler will provide with the necessary information needed for example tap count etc.
 
-**Refer to test scene: Assets\HoloToolkit\Input\Tests\Scenes\InputTapTest.unity**
+**Refer to test scene: Assets\HoloToolkit-Examples\Input\Tests\Scenes\InputTapTest.unity**
 
 ###How to handle the navigation gesture?
 
 1. In your MonoBehavior, implement the interface called INavigationHandler. These events will get raised for navigation events.
-2. The event data inside the event handler will provide with navigation offset to take your action.
+2. The event data inside the event handler will provide navigation offset to take your action.
```

Comment:
I think both versions of this sentence remain unclear. Should this be "The event data inside the event handler provides navigation offsets"? I'm not sure what "to take your action" is trying to mean.